.eslintrc.js

```diff
@@ -369,7 +369,6 @@ module.exports = {
 				'plugin:testing-library/react',
 			],
 			rules: {
-				'testing-library/no-container': 'off',
```

Comment:
This rule is automatically enabled when we include the `testing-library/react` plugin above.